Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -192,7 +192,7 @@ class HistoryPanel: SiteTableViewController, HomePanel {
             make.centerX.equalTo(overlayView)
 
             // Sets proper top constraint for iPhone 6 in portait and for iPad.
-            make.centerY.equalTo(overlayView.snp_centerY).offset(-160).priorityMedium()
+            make.centerY.equalTo(overlayView.snp_centerY).offset(-180).priorityMedium()
```

Comment:
Can we factor this out into a constant somewhere since we have it duplicated here, `RemoteTabsPanel`, and `ReadingListPanel`?